build/commands/lib/config.js

```diff
@@ -280,10 +280,6 @@ Config.prototype.isOfficialBuild = function () {
   return this.isReleaseBuild() && !this.isAsan()
 }
 
-Config.prototype.getBrandingPathProduct = function () {
-  return this.isOfficialBuild() ? "brave" : "brave-development"
```

Comment:
if we no longer need brave-development_behaviors.cc we should remove it

Reply:
cc @goodov

Reply:
we don't have `brave-development_behaviors.cc` in `brave/` directory anywhere. It was copied from `chromium_src/chrome/installer/setup/brave_behaviors.cc` to `src/chrome/.../brave-development_behaviors.cc`.

the copied file does nothing now, feel free to ignore it or just remove manually.